apps/web/next.config.mjs

```diff
@@ -39,7 +39,7 @@ const nextConfig = {
       bodySizeLimit: '25mb',
     },
   },
-  webpack: (config) => {
+  turbo: (config) => {
```

Comment:
This remove a warning in the nextjs logs and looks it's working. We're no longer using Webpack 🎉